sp800-63b/sec9_privacy.md

```diff
@@ -8,22 +8,22 @@
 
-1. The likelihood that the records retention could create a problem for the subscriber such as invasiveness or unauthorized access to the information.
+1. The likelihood that the records retention could create a problem for the subscriber, such as invasiveness or unauthorized access to the information.
 2.  The impact if a problem did occur.
 
-CSPs should be able to reasonably justify any response they take to identified privacy risks, including accepting the risk, mitigating the risk; and sharing the risk. The use of subscriber consent is a form of sharing the risk, and therefore appropriate for use only when a subscriber could reasonably be expected to have the capacity to assess and accept the shared risk.
+CSPs should be able to reasonably justify any response they take to identified privacy risks, including accepting the risk, mitigating the risk, and sharing the risk. The use of subscriber consent is a form of sharing the risk, and therefore appropriate for use only when a subscriber could reasonably be expected to have the capacity to assess and accept the shared risk.
 
 ### 9.2. Privacy Controls
 
 [Section 4.4](#aal_privacy) encourages CSPs to employ appropriately-tailored privacy controls. NIST [[SP 800-53]](#sp800-53) provides a set of privacy controls for CSPs to consider when deploying authentication mechanisms. These controls cover notices, redress, and other important considerations for successful and trustworthy deployments. 
 
 ### 9.3. Use Limitation
 
-[Section 4.4](#aal_privacy) does not permit the CSP to use information about authenticators that is collected and maintained in the authentication process for any purpose other than authentication or to comply with law or legal process, unless the CSP provides clear notice and obtains consent from the subscriber for additional uses. Care should be taken to ensure that use of such information is limited to its original purpose for collection. Consult your SAOP if there are questions about whether proposed agency uses fall within this scope. As stated in [Section 4.4](#aal_privacy), acceptance by the subscriber of additional uses SHALL NOT be a condition of providing authentication services. 
+[Section 4.4](#aal_privacy) does not permit the CSP to use information about authenticators that is collected and maintained in the authentication process for any purpose other than authentication, or to comply with law or legal process, unless the CSP provides clear notice and obtains consent from the subscriber for additional uses. Care should be taken to ensure that the use of such information is limited to its original purpose for collection. Consult your SAOP if there are questions about whether proposed agency uses fall within this scope. As stated in [Section 4.4](#aal_privacy), acceptance by the subscriber of additional uses SHALL NOT be a condition of providing authentication services. 
```

Comment:
I wouldn't add that comma; it makes "or to comply with law or legal process" seem like a nonessential phrase which it isn't.